doc/languages.rst

```diff
@@ -198,6 +198,35 @@ to view the docstring of the syntax checker.  Likewise, you may use
          The cppcheck suppressions file to use via the
          ``--suppressions-list=`` argument.
 
+.. supported-language:: CUDA C/C++
```

Comment:
The list of languages is sorted alphabetically, so this whole section should appear after CSS rather.